packages/flutter_tools/test/general.shard/resident_runner_test.dart

```diff
@@ -892,10 +892,10 @@ void main() {
 
   testUsingContext('ResidentRunner can send target platform to analytics from hot reload', () => testbed.run(() async {
     fakeVmServiceHost = FakeVmServiceHost(requests: <VmServiceExpectation>[
```

Comment:
This fixes were required since I changed the timing of the no-op builds slightly. Fix is to properly await for the app to be started.